scripts/skills/effects/rf_sapling_harvest.nut

```diff
@@ -0,0 +1,45 @@
+this.rf_sapling_harvest <- ::inherit("scripts/skills/skill", {
```

Comment:
Our convention is to append the word `_effect` to every status effect file that goes in the `skills/effects` folder. So please change the filename and the classname here to `rf_sapling_harvest_effect`.